axelor-human-resource/src/main/java/com/axelor/apps/hr/service/expense/ExpenseWorkflowService.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2005-2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.axelor.apps.hr.service.expense;
+
+import com.axelor.apps.base.AxelorException;
+import com.axelor.apps.hr.db.Expense;
+import com.axelor.message.db.Message;
+import java.io.IOException;
+import wslite.json.JSONException;
+
+public interface ExpenseWorkflowService {
+
+  public void confirm(Expense expense) throws AxelorException;
+
+  public Message sendConfirmationEmail(Expense expense)
+      throws AxelorException, ClassNotFoundException, IOException, JSONException;
+
+  public void refuse(Expense expense) throws AxelorException;
+
+  public Message sendRefusalEmail(Expense expense)
+      throws AxelorException, ClassNotFoundException, IOException, JSONException;
+
+  public void cancel(Expense expense) throws AxelorException;
+
+  public Message sendCancellationEmail(Expense expense)
+      throws AxelorException, ClassNotFoundException, IOException, JSONException;
```

Comment:
since we made a service per status, for validate and ventilate, please continue with this logic and create a service per status by splitting this class.